example-doc-generator/python/agent_server.py

```
@@ -0,0 +1,262 @@
+                    "playwright": {
+                        "command": "npx",
+                        "args": [
+                            "@playwright/mcp@latest",
+                            "--headless",
+                            "--viewport-size=1720,968",
+                            f"--output-dir={CWD}/artifacts/screenshots",
+                        ],
+                    }
+                },
+                permission_mode="acceptEdits",
+                max_buffer_size=32 * 1024 * 1024,  # 32 MB — screenshots at 1920x1080 can be 1.5–3 MB as base64
```

Ruff (0.15.7)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)